pkg/core/state/native_state.go

```diff
@@ -157,5 +159,10 @@ func (s *NEOBalance) FromStackItem(item stackitem.Item) error {
 		return fmt.Errorf("invalid public key bytes: %w", err)
 	}
 	s.VoteTo = pub
+	lastGasPerVote, err := structItem[3].TryInteger()
```

Comment:
We can get an out of bounds access here if, for example, a new RPC client is to call `getAccountState` NEO method on an old server that will return just two elements. So to stay compatible you should check `len(structItem) >= 4` before accessing `structItem[3]` (and add a test to decode old structure). Then please add a note to ROADMAP.md and create an issue to remove this compatibility code in half a year from now (when everyone will be upgraded).

Reply:
Done.